runtime/Makefile

```diff
@@ -37,26 +37,32 @@ docker-images:
 	cd base ; docker build --file php-73.Dockerfile -t bref/build-php-73 --target build-environment .
 	cd base ; docker build --file php-74.Dockerfile -t bref/build-php-74 --target build-environment .
 	cd base ; docker build --file php-80.Dockerfile -t bref/build-php-80 --target build-environment .
+	cd base ; docker build --file php-81.Dockerfile -t bref/build-php-81 --target build-environment .
 	# Build the whole Dockerfile to generate the cleaned images that will be used in the next step
 	cd base ; docker build --file php-73.Dockerfile -t bref/tmp/cleaned-build-php-73 .
 	cd base ; docker build --file php-74.Dockerfile -t bref/tmp/cleaned-build-php-74 .
 	cd base ; docker build --file php-80.Dockerfile -t bref/tmp/cleaned-build-php-80 .
+	cd base ; docker build --file php-81.Dockerfile -t bref/tmp/cleaned-build-php-81 .
 	# - function
 	cd layers/function ; docker build -t bref/php-73 --build-arg PHP_VERSION=73 .
 	cd layers/function ; docker build -t bref/php-74 --build-arg PHP_VERSION=74 .
 	cd layers/function ; docker build -t bref/php-80 --build-arg PHP_VERSION=80 .
+	cd layers/function ; docker build -t bref/php-81 --build-arg PHP_VERSION=81 .
 	# - fpm
 	cd layers/fpm ; docker build -t bref/php-73-fpm --build-arg PHP_VERSION=73 .
 	cd layers/fpm ; docker build -t bref/php-74-fpm --build-arg PHP_VERSION=74 .
 	cd layers/fpm ; docker build -t bref/php-80-fpm --build-arg PHP_VERSION=80 .
+	cd layers/fpm ; docker build -t bref/php-81-fpm --build-arg PHP_VERSION=81 .
 	# - console
 	cd layers/console ; docker build -t bref/php-73-console --build-arg PHP_VERSION=73 .
 	cd layers/console ; docker build -t bref/php-74-console --build-arg PHP_VERSION=74 .
 	cd layers/console ; docker build -t bref/php-80-console --build-arg PHP_VERSION=80 .
+	cd layers/console ; docker build -t bref/php-81-console --build-arg PHP_VERSION=81 .
 	# Other Docker images
 	cd layers/fpm-dev ; docker build -t bref/php-73-fpm-dev --build-arg PHP_VERSION=73 .
 	cd layers/fpm-dev ; docker build -t bref/php-74-fpm-dev --build-arg PHP_VERSION=74 .
 	cd layers/fpm-dev ; docker build -t bref/php-80-fpm-dev --build-arg PHP_VERSION=80 .
+	#cd layers/fpm-dev ; docker build -t bref/php-81-fpm-dev --build-arg PHP_VERSION=81 . # xdebug not ok
```

Comment:
Would have been nice to enabling it but without xdebug ATM, WDYT? I can try a PR if :ok:

Reply:
👍  sounds good to me

Reply:
I will try to rework on it if my free time allows me, go :+1: for trying on your side

Reply:
Whoever picks this up, consider taking a quick look at https://github.com/brefphp/bref/blob/master/Makefile#L18 to see if it's easy to start publishing php-81 images on Docker Hub, that would help me test PHP 8.1!

Reply:
@deleugpn @mykiwi @mnapoli the PR is here :wink: 
https://github.com/brefphp/bref/pull/1007

@Nyholm I also have started some changes in extra extensions repository so will make a PR once php-81 docker images are published on docker hub.

Reply:
Sorry #1008 as an upfix of #1007... but I've also prepared the extra extensions PR https://github.com/brefphp/extra-php-extensions/pull/261